Explain prerequisites for ETL large memory tweaks option (#4906)
centreon/centreon-documentation: versioned_docs/version-25.10/reporting/installation.md

| **General options**                                                                    |                                                                                                                                                                                                                                       |
| Reporting engine uses a dedicated dedicated MySQL server                             | The only appropriate option is **Yes**.                                                                                                                                                                                     |
| Temporary file storage directory on reporting server                                   | Folder where dumps will be stored on the reporting server                                                                                                                                                                             |

> | Use large memory tweaks (store MariaDB temporary tables in memory)                     | Creates temporary tables using RAM rather than creating them on disk. Not recommended to users have databases with less than 64GB of RAM.                                                  |

Reviewer: ```suggestion
| Use large memory tweaks (store MariaDB temporary tables in memory)                     | Creates temporary tables using RAM rather than creating them on disk. Not recommended if you have databases with less than 64GB of RAM.                                                  |
```

> | Use large memory tweaks (store MariaDB temporary tables in memory)                     | Creates tables using RAM rather than creating them on disk. Not recommended to users who do not have databases with large amounts of RAM (+64GB)                                                  |

Author: ```suggestion
| Use large memory tweaks (store MariaDB temporary tables in memory)                     | Creates temporary tables using RAM rather than creating them on disk. Not recommended to users have databases with less than 64GB of RAM.                                                  |
```
